cmd/create-index-snapshot/main.go

```diff
@@ -29,7 +29,16 @@ import (
 	"github.com/optakt/flow-dps/service/dictionaries"
 )
 
+const (
+	success = 0
+	failure = 1
+)
+
 func main() {
+	os.Exit(run())
+}
+
+func run() int {
```

Comment:
Why do this everywhere? How is it cleaner than to just do `os.Exit(failure)` in `func main()` or using `log.Fatal()`?